test-cases/tests/alerts/c05b-verify-alerts-are-exposed-and-no-critical-alerts-have-fired.md

```diff
@@ -6,6 +6,8 @@ products:
     environments:
       - osd-fresh-install
       - osd-post-upgrade
+    targets:
+    - 1.29.0
```

Comment:
All modifications like this (adding the whole `target` flag) need to be reverted. There's already another tag ( `per-build` in this particular case) that causes that the test case is generated when needed - for each rc in this particular case. No need to use `target`.